src/main/java/TaskList.java

```diff
@@ -0,0 +1,84 @@
+import java.io.IOException;
+
+public class TaskList {
+    Task[] taskList = new Task[100];
+    // The first index that is empty
+    private int firstEmptyIndex = 0;
+
+    public void addTask(String input, Task.TaskType taskType, boolean isDone) throws IOException {
+        Task task = taskType == Task.TaskType.ToDo
+                ? new Todo(input, isDone)
+                : taskType == Task.TaskType.Event
+                ? new Event(input, isDone)
+                : new Deadline(input, isDone);
+
+        taskList[firstEmptyIndex] = task;
+        DataReadWriter.saveAddedTask(task);
+
+        String msg = "Got it. I've added this task:\n" + "  "
+                + taskList[firstEmptyIndex].toString()
+                + "\n" + "Now you have " + (firstEmptyIndex + 1) + " tasks in the list";
+        System.out.println(msg);
+        ++firstEmptyIndex;
+    }
+
+    public void appendLoadedTask(Task task) throws IOException {
+
+        taskList[firstEmptyIndex] = task;
+        ++firstEmptyIndex;
+    }
+
+    public void markTaskAsDone(int taskNum) {
+        if (taskNum >= firstEmptyIndex) {
+            System.out.println("There is no task with index " + (taskNum + 1));
+            return;
+        }
+        taskList[taskNum].markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(taskList[taskNum].toString());
+    }
+
+    public void markTaskAsUnDone(int taskNum) {
+        if (taskNum >= firstEmptyIndex) {
+            System.out.println("There is no task with index " + (taskNum + 1));
+            return;
+        }
+        taskList[taskNum].markAsUnDone();
+        System.out.println("I've marked this task as not done:");
+        System.out.println(taskList[taskNum].toString());
+    }
+
+    public void deleteTask(int index) {
+        if (index >= firstEmptyIndex) {
+            System.out.println("There is no task with index " + (index + 1));
+            return;
+        }
+        System.out.println("Noted. I've removed this task:\n" +
```

Comment:
The line break should be placed before the operator.
<img width="725" alt="image" src="https://user-images.githubusercontent.com/60882425/187225273-e978c2de-7e31-4b4c-9a86-9e0866965ee7.png">